### ISSUE TITLE
Create app missing some of the shape types for particles.

### ISSUE THREAD

**Issue by 1st-BrainStormer** [vircadia/vircadia#492]
Particles should have the same shape types as shape entities.  These shapes as well as compound shapes are used to define where particles can be generated from either the surface to the volume of the shape.

**Comment by HifiExperiments** [comment]
This is a little more complicated than just exposing them in create.  Currently, particles only support box, cylinder, circle, plane, sphere, ellipsoid, and compound.  For any other shapes we want to expose, there's a bit of math to be done to correctly sample spawn points from within the shape.

**Comment by 1st-BrainStormer** [comment]
I suppose the other shape types could just be loaded as a compound shapes in the background if we wanted more options in the create app.

**Comment by AleziaKurdis** [comment]
"Compound" seems to be the workaround for now.

For the Create App. I would recommend following what the API really does. 

**API Side enhancement**

**Comment by AleziaKurdis** [comment]
Not sure it even worth the investment. 
For sure, this should be very very low in the priority list.

**Comment by daleglass** [comment]
Yeah, I was mostly checking if we closed anything we shouldn't have. There's been some interest in better particles, so figured might as well keep this one.
